plugins/@grouparoo/pipedrive/src/lib/client.ts

```diff
@@ -67,22 +54,29 @@ export class PipedriveClient {
     return data;
   }
 
-  async deletePerson(id: number): Promise<Record<string, any>> {
-    const { data } = await this.request.delete(`/persons/${id}`);
+  async delete(
+    id: number
+  ): Promise<Record<string, any>> {
```

Comment:
Do we know what these values actually are, or is it based on the users' pipedrive setup?

Reply:
In all cases, `persons.delete` return value isn't used!
Not sure if it's created like that for a reason.